test/test_datasets_utils.py

```diff
@@ -214,5 +216,30 @@ def test_verify_str_arg(self):
         pytest.raises(ValueError, utils.verify_str_arg, "b", ("a",), "arg")
 
 
+class TestFolderUtils:
```

Comment:
There's only one method, maybe we could just have a single function instead of a class?